src/Server/Resource/DoctrineResource.php

```diff
@@ -348,6 +348,11 @@ public function create($data)
 
         $this->getObjectManager()->flush();
 
+        $results = $this->triggerDoctrineEvent(DoctrineResourceEvent::EVENT_CREATE_FLUSH, $entity, $data);
+        if ($results->last() instanceof ApiProblem) {
```

Comment:
I could change it, but I'm not sure why we should limit.

Secondly - could you see potential issue here? I think if somebody returns ApiProblem here, he knows what he is doing... For example you could create an entity (ie. photo) and next upload the image somewhere... and on error you could throw an ApiProblem // yeah, I know it's not the best idea, just an example.

Another thing - I'm trying to revert your change (using new event instead). There was the same `ApiProblem` logic.